content/docs/reference/addon-api/addon.tab/_index.md

```diff
@@ -205,34 +211,10 @@ Options `condition`, `reduxCondition` and `reduxEvents` should be used as optimi
     <td>Yes</td>
     <td>Element to hide</td>
   </tr>
-  <tr>
-    <td>options</td>
-    <td><code>Object</code></td>
-    <td>No</td>
-    <td>
-      <table>
-        <tr>
-          <th>Property</th>
-          <th>Type</th>
-          <th>Required</th>
-          <th>Default</th>
-          <th>Description</th>
-        </tr>
-        <tr>
-          <td>display</td>
-          <td><code>String</code></td>
-          <td>No</td>
-          <td><code>""</code></td>
-          <td>The <code>display</code> CSS value to use when the addon is enabled.</td>
-        </tr>
-      </table>
-    </td>
-  </tr>
 </table>
 
 Hides the given element with `display: none` when the addon is disabled, until it is reenabled.  
-If the intended `display` CSS property value for the provided element when visible is not the default value for the type of provided element (for example, `block` for `div`s and `inline` for `span`s), you should provide that value inside the options parameter.  
-If you want to manually hide the element in situations where the addon is enabled, you should use a dedicated class name for that, instead of manually setting `el.style.display = "none";`. Use a class name selector in a userstyle to set `display: none !important;` on the element.
+If you want to manually hide the element in situations where the addon is enabled, you should use a dedicated class name for that, instead of manually setting `el.style.display = "none";`. Use a class name selector in a userstyle to set `display: none;` on the element.

```

Comment:
Is this still a requirement or just best practice?

Reply:
It's not a requirement because that addon API doesn't change style attribute anymore.

Reply:
I guess the example near the top of the page isn't needed any more then?

Reply:
Yes, I think the example can be removed.